### PR TITLE
Update manage-dynamic-distribution-groups.md

### PR DESCRIPTION
**Pull request by JCrosbyMSFT**
Added warning that DDLs are not synced back from EXO to Azure/Onprem

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Exchange/ExchangeOnline/recipients-in-exchange-online/manage-dynamic-distribution-groups/manage-dynamic-distribution-groups.md [hunk 1]
```diff
@@ -29,6 +29,9 @@ Unlike regular distribution groups that contain a defined set of members, the me
 > [!IMPORTANT]
 > A dynamic distribution group includes any recipient in Active Directory with attribute values that match its filter. If a recipient's properties are modified to match the filter, the recipient could inadvertently become a group member and start receiving messages that are sent to the group. Well-defined, consistent account provisioning processes will reduce the chances of this issue occurring.
 
+> [!IMPORTANT]
+> Dynamic distribution groups are not synced from Exchange Online to Azure Active Directory or to your onpremises Active Directory. Therefore features such as Azure Conditional Access does not support being scoped to an Exchange Online dynamic distribution group.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/recipients-in-exchange-online/manage-dynamic-distribution-groups/manage-dynamic-distribution-groups.md:33]:
```suggestion
> - A dynamic distribution group includes any recipient in Active Directory with attribute values that match its filter. If a recipient's properties are modified to match the filter, the recipient could inadvertently become a group member and start receiving messages that are sent to the group. Well-defined, consistent account provisioning processes will reduce the chances of this issue occurring.
> - Dynamic distribution groups are not synced from Exchange Online to Azure Active Directory or to your on-premises Active Directory. Therefore, features such as Azure Conditional Access do not support being scoped to an Exchange Online dynamic distribution group.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review. @JCrosbyMSFT - kindly commit my suggested changes if you agree with them.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@Kateyanne PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**